packages/spark-core/_spark-core.scss

```diff
@@ -45,7 +46,12 @@
 @import './tools/animations';
 @import './tools/functions';
 @import './tools/mixins';
-@import './base/inputs';
+@import 'base/inputs/error-container';
```

Comment:
Thinking maybe we should have a manifest for all the inputs so we don't have to import them all separately like this (since core stuff is made to go together anyway).